### PR TITLE
CLI changes for JsComposition

### PR DESCRIPTION
**Pull request by mrtanishqbhatia**
This PR contains the following changes:

1. CLI for JsComposition basic
2. Lint fixes and Make network and level case insensitive
3. Fix to postinstall script

### CODE REVIEW

**Comment by soniadimitrov** [comment]
Please hold on merge until there is confirmation from Luca on the release of Pulsar related changes ~ Aug. 26.

File: package.json [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-  "version": "1.8.1",
+  "version": "1.9.0",
   "description": "A tool that makes it easier to manage Akamai EdgeWorkers code bundles and EdgeKV databases. Call the EdgeWorkers and EdgeKV API from the command line.",
   "repository": "https://github.com/akamai/cli-edgeworkers",
   "scripts": {
     "build": "npm run build-ts",
     "clean": "rm -rf node_modules",
     "build-ts": "tsc",
-    "postinstall": "if test \"$NODE_ENV\" != \"production\" ; then npm run build ; fi",
+    "postinstall": "node scripts/postinstall",
```
Comment (kyleli-akamai) [inline package.json:10]: We might need to change this to
```suggestion
    "postinstall": "node scripts/postinstall.js",
```

**Review by kyleli-akamai — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (soniadimitrov) [reply package.json:10]: @kyleli-akamai , please update in our internal mirror.

Comment (kyleli-akamai) [reply package.json:10]: Sure.

**Review by kyleli-akamai — APPROVED** [review]

**Pull request merged**